packages/amazonq/src/lsp/chat/webviewProvider.ts

```diff
@@ -12,9 +12,10 @@ import {
     WebviewViewResolveContext,
     Uri,
 } from 'vscode'
-import { LanguageServerResolver } from 'aws-core-vscode/shared'
 import { QuickActionCommandGroup } from '@aws/mynah-ui'
 import * as path from 'path'
+import { LanguageServerResolver } from 'aws-core-vscode/shared'
+import { disclaimer } from 'aws-core-vscode/amazonq'
```

Comment:
Importing this index file pulls in a lot of code that currently doesn't have web support.

We can avoid this anyway, because `settings.ts` is intended to have application-specific settings logic in it, thus `disclaimer.ts` isn't needed.